content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -567,7 +561,7 @@ Follow the steps given below to rollback the Deployment from the current version
     CreationTimestamp:      Sun, 02 Sep 2018 18:17:55 -0500
     Labels:                 app=nginx
     Annotations:            deployment.kubernetes.io/revision=4
-                            kubernetes.io/change-cause=kubectl set image deployment.v1.apps/nginx-deployment nginx=nginx:1.16.1 --record=true
+                            kubernetes.io/change-cause=kubectl set image deployment.v1.apps/nginx-deployment nginx=nginx:1.16.1
```

Comment:
Does this annotation definitely get set? What sets it?

Reply:
@sftim
The rollout command is not shown in Annotation without using the `--record` flag.
Only the  `deployment.kubernetes.io/revision=4 ` is set in the annotation